### ISSUE TITLE
cerevisiae with radiation simulation is possible?

### ISSUE THREAD

**Issue by nightandweather** [SysBioChalmers/yeast-GEM#322]
### Description of the issue:

Hi everyone, I have a question. The question may sound strange because I'm not a biology major. Is the result of the radiation irradiation on the cerevisiae and the model results are on caffeine? If not, can the metabolisms of cerevisiae be randomly deactivated due to radiation and the result there of be represented by code?
#### Current feature/value/output:
nope

#### Reproducing these results:
nope

**I hereby confirm that I have:**
- [ ] Tested my code with [all requirements](https://github.com/SysBioChalmers/yeast-GEM#required-software---user) for running the model
- [ ] Done this analysis in the `main` branch of the repository
- [ ] Checked that a similar issue does not exist already
- [ ] If needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/yeast-GEM) about the issue

**Comment by edkerk** [comment]
No, such effects are not captured by the model, unless one would manually expand the model to somehow consider it.
